src/sentry/rules/actions/notify_event_service.py

```diff
@@ -29,8 +29,16 @@ def __init__(self, *args, **kwargs):
 
 
 class NotifyEventServiceAction(EventAction):
-    form_cls = NotifyEventServiceForm
-    label = 'Send a notification via {service}'
+    def __init__(self, *args, **kwargs):
+        super(NotifyEventServiceAction, self).__init__(*args, **kwargs)
+        self.label = 'Send a notification via {service}'
+        self.form_cls = NotifyEventServiceForm
```

Comment:
Why move these two into the instance init?

Reply:
No particular reason, will move back